### PR TITLE
Adjust load data tests to a new test data structure

### PR DESCRIPTION
**Pull request by arkid15r**
## Issue Number

https://github.com/AlexsLemonade/scpca-portal/issues/447

## Purpose/Implementation Notes

Refactor tests to use the new single project test data structure.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/scpca_portal/test/management/commands/test_load_data.py [hunk 1]
```diff
@@ -386,34 +395,36 @@ def test_single_cell_metadata(self):
         sample_metadata_keys = sample_metadata_lines[0].split(common.TAB)
         self.assertEqual(sample_metadata_keys, expected_keys)
 
-        # There are 4 files for each sample, plus a README.md
-        # and a single_cell_metadata.tsv file.
-        # 6 = 1 * 4 + 2
-        self.assertEqual(len(project_zip.namelist()), 6)
+        # There are 4 files for each sample, plus a README.md,
+        # a single_cell_metadata.tsv file, and 2 bulk_ files.
+        # 8 = 1 * 4 + 4
```
Comment (davidsmejia) [inline api/scpca_portal/test/management/commands/test_load_data.py:400]:
```suggestion
        # There are 4 files for each sample, plus a README.md,
        # a single_cell_metadata.tsv file, and 2 bulk_ files.
        # A project with one sample should contain 8 files:
        # ex: 
        # ├── README.md
        # ├── SCPCS000001
        # │   ├── SCPCL000001_filtered.rds
        # │   ├── SCPCL000001_processed.rds
        # │   ├── SCPCL000001_qc.html
        # │   └── SCPCL000001_unfiltered.rds
        # ├── bulk_metadata.tsv
        # ├── bulk_quant.tsv
        # └── single_cell_metadata.tsv
```

I was having difficulty understanding this, I think we could either rework the wording or just add in an example for clarity.

**Review by davidsmejia — COMMENTED** [review]
Looking good to me. I had one suggestion to improve a comment.

File: api/scpca_portal/test/management/commands/test_load_data.py (hunk 1 above)
Comment (arkid15r) [reply api/scpca_portal/test/management/commands/test_load_data.py]: That's a nice visualization example! I added similar description to all modality/data outputs.

**Review by arkid15r — COMMENTED** [review]
PTAL

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**